src/DurableFunctions.CodeGeneration/WebJobs.Extensions.DurableTask.CodeGen.sln

```diff
@@ -0,0 +1,43 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.31205.134
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DurableFunctions.CodeGen.Example", "Example\DurableFunctions.CodeGen.Example.csproj", "{0C0D1182-4069-4895-9D5D-6C8A6C4D8E25}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DurableFunctions.CodeGen.SourceGenerator", "SourceGenerator\DurableFunctions.CodeGen.SourceGenerator.csproj", "{66580511-CAB3-4CDF-9A56-3C8F0380CE80}"
```

Comment:
Why do we need to call this SourceGenerator? This is one project, it should be just CodeGen/CodeGenerator/SourceGenerator etc.

Reply:
So SourceGenerator is the name of the .NET technology used for the code generation, so that is why I kept that in the name. Not sure I quite understand the second pont?